README.md

```diff
@@ -0,0 +1,160 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Add Food]**
+- **[Add Recipe]**
+- **[Add Inventory]**
+- **[User Authentication]**
+- **[View Food and Recipe]**
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need the following installed in your local system
+
+<ul>
+<li>Ruby</li>
+<li>Git CLI</li>
+<li>Github account</li>
+</ul>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/andrianarivo/recipe_app.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd recipe_app
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  ruby main.rb
+```
+### Run tests
+
+-  Type `Rspec`: To run test `
+-  Type `npx stylelint "**/*.{css,scss}"` to run for stylelint errors
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Andrianariv David**
+
+- GitHub: [@andrianarivo](https://github.com/andrianarivo)
+- Twitter: [@andrianarivo](https://twitter.com/dandrianarivo)
+- LinkedIn: [@andrianarivo](https://www.linkedin.com/in/andrianarivo)
+
+👤 **Racha Ould Babaali**
+
+- GitHub: [@rachaouldbabaali](https://github.com/rachaouldbabaali)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/rachaouldbabaali/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Build the front end of the system]**
+- [ ] **[Style the front end]**
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/andrianarivo/recipe_app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please leave a STAR to the [Github repo](https://github.com/andrianarivo/recipe_app/tree/dev)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to appreciate [Microverse](https://www.microverse.org/) for providing the resources and the platform for us to be a Professional full-stack developer.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Kindly fix the link to your MIT License page it is currently broken it shows a 404 error page 👍🏿